rest/oidc_api.go

```diff
@@ -99,9 +113,44 @@ func (h *handler) handleOIDCCommon() (redirectURLString string, err error) {
 		return redirectURLString, err
 	}
 
+	if !provider.IsDefault {
+		base.Debugf(base.KeyAuth, "Adding provider (%v) to callback URL", base.UD(provider.Name))
+		if err = addCallbackURLQueryParam(redirectURL, oidcAuthProvider, provider.Name); err != nil {
+			base.Errorf("Failed to add provider to callback URL, err: %v", err)
+		}
+		base.Debugf(base.KeyAuth, "Callback URL: %s", redirectURL.String())
```

Comment:
Feels like more dev-time (trace) logging. If you disagree, let's just reduce the amount of debug level logging we're doing to only 1 line by combining both.

```suggestion
		base.Tracef(base.KeyAuth, "Callback URL: %s", redirectURL.String())
```

Reply:
Make sense. Fixed in latest revision.